src/replace-libraries-build-builder.ts

```diff
@@ -0,0 +1,34 @@
+import {versions} from './angular-versions';
+import {AngularJson} from './types/angular-json';
+
+export const pre10_1LibraryBuilder = '@angular-devkit/build-ng-packagr:build';
+export const post10_1LibraryBuilder =
+  '@angular-devkit/build-angular:ng-packagr';
+
+export function replaceLibrariesBuildBuilder(
+  angularVersion: string,
+  angularJson: AngularJson
+): AngularJson {
+  const modifiedAngularJson = {...angularJson};
+  for (const key in angularJson.projects) {
+    if (Object.prototype.hasOwnProperty.call(angularJson.projects, key)) {
+      const project = angularJson.projects[key];
+
+      if (project.projectType === 'library' && !!project.architect.build) {
```

Comment:
suggestion: Rather than looking for a `build` target, look for the specific ng-packagr builders. People might have renamed the `build` target to for example `build-lib` or something else entirely. The name is not what matters.

Reply:
Interesting, yeah that makes sense.